From 1f45f6be52ea34a4994709b72775242bcbf79f39 Mon Sep 17 00:00:00 2001
From: Ryan Gonzalez <rymg19@gmail.com>
Date: Thu, 30 Sep 2021 11:27:43 -0500
Subject: [PATCH] Clang build script: Disable hwasan

hwasan doesn't seem to actually compile if not on a host x86-64 system,
and we don't really need this for the Flatpak anyway.
---
 tools/clang/scripts/build.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/clang/scripts/build.py b/tools/clang/scripts/build.py
index 535d1ef55e5d7..e43741386d746 100755
--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -593,7 +593,7 @@ def compiler_rt_cmake_flags(*, sanitizers, profile):
       'COMPILER_RT_BUILD_SANITIZERS=' + ('ON' if sanitizers else 'OFF'),
       'COMPILER_RT_BUILD_XRAY=OFF',
       # See crbug.com/1205046: don't build scudo (and others we don't need).
-      'COMPILER_RT_SANITIZERS_TO_BUILD=asan;dfsan;msan;hwasan;tsan;cfi',
+      'COMPILER_RT_SANITIZERS_TO_BUILD=asan;dfsan;msan;tsan;cfi',
       # We explicitly list all targets we want to build, do not autodetect
       # targets.
       'COMPILER_RT_DEFAULT_TARGET_ONLY=ON',
-- 
2.46.0

